### ISSUE TITLE
Identify simple experiment

### ISSUE THREAD

**Issue by chrisparnin** [alt-code/AutoSpark#46]
Find a simple dataset that should take at least an hour.

**Comment by amritbhanu** [comment]
Just an update, we can run our original experiments on spark if we have the environment and all the needed packages installed on the machine. Just ran the experiment which are written in python on my local machine on spark.

So, if we can install the needed packages on the instances, we don't need to change our python code to scala.

**Comment by chrisparnin** [comment]
Yes spark works with a lot of backends. But to truly take advantage of the infrastructure, you have to have jobs use spark constructs...e.g.:

```python
# Import SQLContext and data types
from pyspark.sql import SQLContext
from pyspark.sql.types import *

# sc is an existing SparkContext.
sqlContext = SQLContext(sc)

# Load a text file and convert each line to a tuple.
lines = sc.textFile("examples/src/main/resources/people.txt")
parts = lines.map(lambda l: l.split(","))
people = parts.map(lambda p: (p[0], p[1].strip()))

# The schema is encoded in a string.
schemaString = "name age"

fields = [StructField(field_name, StringType(), True) for field_name in schemaString.split()]
schema = StructType(fields)

# Apply the schema to the RDD.
schemaPeople = sqlContext.createDataFrame(people, schema)

# Register the DataFrame as a table.
schemaPeople.registerTempTable("people")

# SQL can be run over DataFrames that have been registered as a table.
results = sqlContext.sql("SELECT name FROM people")

# The results of SQL queries are RDDs and support all the normal RDD operations.
names = results.map(lambda p: "Name: " + p.name)
for name in names.collect():
  print(name)
```

**Comment by amritbhanu** [comment]
@gohilankit can you run this [code] (https://github.com/alt-code/AutoSpark/tree/master/Amrit/01-14) on AWS once it is set up? Let me know about the result and its runtime.

In the meantime I am changing our original rig of text mining in PySpark form for further use.

**Comment by gohilankit** [comment]
@amritbhanu The zip archive seems to be corrupt. I'm not able to extract it. Can you upload the text file again?

Archive:  jones.txt.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.

**Comment by amritbhanu** [comment]
@gohilankit I uploaded the zip file on drive. Please download it from here. Here is the [link] (https://drive.google.com/file/d/0B6L_ShRNMys7c1RVbVQxcjBXMGc/view?usp=sharing)

Keep the extracted file and the running code in the same directory.

**Comment by chrisparnin** [comment]
@amritbhanu Can you get your "shingling" experiment and hand-off to @gohilankit

**Comment by amritbhanu** [comment]
@gohilankit read these instructions for running shingling results [here] (https://github.com/alt-code/AutoSpark/tree/master/Amrit/01-22)
